src/tools/align/components/results/AlignResult.tsx

```diff
@@ -109,30 +102,35 @@ const useParamsData = (
   return paramsData;
 };
 
+type Params = {
+  id: string;
+  subPage?: TabLocation;
+};
+
 const AlignResult = () => {
   const history = useHistory();
-  const match = useRouteMatch(LocationToPath[Location.AlignResult]) as Match;
+  const match = useRouteMatch<Params>(LocationToPath[Location.AlignResult]);
 
   const [selectedEntries, setSelectedEntries] = useState<string[]>([]);
 
   // if URL doesn't finish with "overview" redirect to /overview by default
   useEffect(() => {
-    if (!match.params.subPage) {
+    if (!match?.params?.subPage) {
       history.replace(
         history.createHref({
           ...history.location,
           pathname: `${history.location.pathname}/${TabLocation.Overview}`,
         })
       );
     }
-  }, [match.params.subPage, history]);
+  }, [match, history]);
```

Comment:
Well, it would be called more times (every time the subPage changes). But the first thing is an if to not do anything if a subPage is present. Thing is, match might be undefined (if we use the align result component in the wrong place), so I need to check match, as it will cause an error to check match.param.subPage if match is undefined.